roles/network_plugin/calico/tasks/main.yml

```diff
@@ -8,13 +8,14 @@
     mode: 0644
   notify:
     - restart docker
+  when: ansible_os_family != "CoreOS"
 
 - name: Calico | Write docker.service systemd file
   template:
     src: systemd-docker.service
     dest: /lib/systemd/system/docker.service
   notify: restart docker
-  when: ansible_service_mgr == "systemd"
+  when: ansible_service_mgr == "systemd" and ansible_os_family != "CoreOS"
```

Comment:
currently we don't need to change anything in the default docker config on CoreOS